be/app/services/exerciseService.js

```diff
@@ -0,0 +1,53 @@
+const db = require('../models')
+const ExerciseTraining = db.exercise_training
+const Exercise = db.exercise
+
+class ExerciseService {
+  static async getExerciseOveral(trainingPlanId) {
+    try {
+      const trainingSessions = await ExerciseTraining.find({ training_plan_id: trainingPlanId })
+      const dayMap = new Map()
+
+      for (const session of trainingSessions) {
```

Comment:
(Không bắt buộc) T nghĩ nên đổi thành
```suggestion
await Promise.all(trainingSessions.map(async (session) => {
  const { day, exercise_id } = session
  ...
}))
```